prisma/schema.prisma

```diff
@@ -79,10 +79,10 @@ model Redemption {
   id_details       Json?
   age              Int?
   failure_message  String?   @db.VarChar
```

Comment:
Allows us to reuse Pools constant that is shared with `AIRDROP_CONFIG`

Reply:
That's fine, though the older names were way better